lib/generators/USAGE

```diff
@@ -0,0 +1,125 @@
+
+* Bootstrap Integration
+
+    React on Rails ships with Twitter Bootstrap already integrated into the build.
+    Note that the generator removes require_tree in both the application.js and
+    application.css.scss files. This is to ensure the correct load order for the
+    bootstrap integration, and is usually a good idea in general. You will therefore
+    need to explicitly require your files.
+
+    How the Bootstrap library is loaded depends upon whether one is using the Rails
+    server or the HMR development server.
+
+    1. Bootstrap via Rails Server
+
+        In the former case, the Rails server loads bootstrap-sprockets, provided
+        by the bootstrap-sass ruby gem (added automatically to your Gemfile by
+        the generator) via the `app/assets/stylesheets/_bootstrap-custom.scss`
+        partial.
+
+        This allows for using Bootstrap in your regular Rails stylesheets. If you
+        wish to customize any of the Bootstrap variables, you can do so via the
+        `client/assets/stylesheets/_pre-bootstrap.scss` partial.
+
+    2. Bootstrap via Webpack Dev Server
+
+        When using the webpack dev server, which does not go through Rails,
+        bootstrap is loaded via the `bootstrap-sass-loader` which uses the
+        `client/bootstrap-sass-config.js` file.
+
+    3. Keeping Custom Bootstrap Configurations Synced
+
+        Because the webpack dev server and Rails each load Bootstrap via a different
+        file (explained in the two sections immediately above), any changes to
+        the way components are loaded in one file must also be made to the other
+        file in order to keep styling consistent between the two. For example,
+        if an import is excluded in _bootstrap-custom.scss, the same import should
+        be excluded in `bootstrap-sass-config.js` so that styling in the Rails
+        server and the webpack dev server will be the same.
+
+    4. Skip Bootstrap Integration
+
+        Bootstrap integration is enabled by default, but can be disabled by passing
+        the --skip-bootstrap flag (alias -b). When you don't need Bootstrap in your
+        existing project, just skip it as needed.
+
+* Linters
+
+    The React on Rails generator can add linters and their recommended accompanying
+    configurations to your project. There are two classes of linters: ruby linters
+    and JavaScript linters.
+
+* JavaScript Linters
+
+    JavaScript linters are enabled by default, but can be disabled by passing the
+    --skip-js-linters flag (alias j), and those that run in Node have been add to
+    `client/package.json` under devDependencies.
+
+* Ruby Linters
+
+    Ruby linters are disabled by default, but can be enabled by passing the
+    `--ruby-linters` flag when generating. These linters have been added to your
+    Gemfile in addition to the the appropriate Rake tasks.
+
+    We really love using all the linters! Give them a try.
+
+* Running the Linters
```

Comment:
I don't think we should include linting information here, this is basically just parroting what the `rake -T lint` command does, and is therefore adding a high maintenance cost without providing any new information. Running the linters is also an entirely different subject than running the generator and should not go in the generator's description.